internal/operator-controller/applier/helm.go

```diff
@@ -53,9 +56,36 @@ type Preflight interface {
 	Upgrade(context.Context, *release.Release) error
 }
 
+type RestConfigMapper func(context.Context, client.Object, *rest.Config) (*rest.Config, error)
+
+type NewForConfigFunc func(*rest.Config) (authorizationv1client.AuthorizationV1Interface, error)
+
+type AuthClientMapper struct {
```

Comment:
```suggestion
type AuthorizationClientMapper struct {
```

Reply:
Haha, yeah I have that in a local change I need to push up, got myself mixed up